src/test/resources/data/gaugePayload.json

```diff
@@ -67,11 +66,20 @@
       "gauge_style":"half",
       "filter":"",
       "gauge_max":"300",
+      "use_kibana_indexes":true,
+      "hide_last_value_indicator":true,
+      "index_pattern":{
+        "id":"${indexPatternId}"
+      },
       "tooltip_mode":"show_all"
     }
   ],
   "state":{
 
   },
-  "savedObjectId":"unsaved"
+  "searchSession":{
+    "sessionId":"e0ee9fb0-6a93-455d-b1e1-4e05dc48929a",
+    "isRestore":false,
+    "isStored":false
+  }
```

Comment:
How relevant is this change?

Reply:
Very relevant! Without this change, it would essentially be as if search sessions feature is disabled.